core/domain/topic_domain.py

```diff
@@ -29,6 +29,7 @@
 
 CMD_CREATE_NEW = 'create_new'
 CMD_CHANGE_ROLE = 'change_role'
+CMD_REMOVED_MANAGER_ROLE = 'removed_manager_role'
```

Comment:
Shouldn't the command be "remove_manager_role" (i.e. an action)?

Reply:
Yeah, changed!